docs/source/advanced/jit.rst

```diff
@@ -206,6 +206,29 @@ A simple constant optimization example::
     The XLA backend (the interface to the hardware) will do the constant optimization and may take a long time and
     a lot of memory due to compilation, often with very little gain in final performance, if any.
 
+RandomState and Jit
+^^^^^^^^^^^^^^^^^^^
+
+You need to pass :code:`objax.random.DEFAULT_GENERATOR` as a variable to :code:`VarCollection`,
```

Comment:
User may use custom generator instead of default, we should mention this as well.

Reply:
updated